tests/v2/bad/wrong-xmlid-value.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="file:/home/toms/repos/GH/opensuse/geekodoc/geekodoc/rng/5.1_1/xmlid-test.rnc" type="application/relax-ng-compact-syntax"?>
```

Comment:
Not sure if you really want the name of your home dir in there.

Reply:
Yes, was a accident and shouldn't be committed. Removed in 197ce8f.